lib/customer.rb

```diff
@@ -0,0 +1,41 @@
+require 'csv'
+
+class Customer
+  attr_reader :id
+  attr_accessor :email, :address
+
+  def initialize (id, email, address)
```

Comment:
Avoid the space between the name and the parenthesis:

```suggestion
  def initialize(id, email, address)
```

I noticed this due to this warning:
```
/Users/kaida/Developer/ada/cohort-13/grocery-store/lib/customer.rb:7: warning: parentheses after method name is interpreted as an argument list, not a decomposed argument
```